etc/generate-package-yml-files-from-PackageInfo-files.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# How to use this:
+# - Edit the gapTarballURL variable. Point it to the tarball of a gap release.
```

Comment:
I don't like that, it doesn't fit in how the release process will work: there, we'll have the tarball already available. In fact, we might have an extracted version.

So what I really want is a script that I can point at the `pkg` dir and then run it. I.e. basically the last few lines of this script

Reply:
Will change it accordingly.